pkg/virt-controller/services/template.go

```diff
@@ -1044,8 +1048,8 @@ func getNamespaceAndNetworkName(vmi *v1.VirtualMachineInstance, fullNetworkName
 func getNetworkToResourceMap(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (networkToResourceMap map[string]string, err error) {
 	networkToResourceMap = make(map[string]string)
 	for _, network := range vmi.Spec.Networks {
-		if network.Multus != nil {
-			namespace, networkName := getNamespaceAndNetworkName(vmi, network.Multus.NetworkName)
+		if multusOrNpwgNetworkName := util.GetMultusOrNpwgNetworkName(network); multusOrNpwgNetworkName != "" {
```

Comment:
this isn't the same logic as before. We're checking if network name exists rather if the struct is set.  This probably wouldn't matter once the validating webhook logic I pointed out gets fixed.

Reply:
done